packages/app-desktop/gui/NoteEditor/NoteBody/PlainEditor/PlainEditor.tsx

```diff
@@ -1,14 +1,76 @@
-
```

Comment:
now that we have established that this wasn't the right file to change, please revert the changes in this file

Reply:
reverted the change in latest commit